instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -6,8 +6,7 @@ plugins {
 base.archivesName.set("opentelemetry-spring-boot")
 group = "io.opentelemetry.instrumentation"
 
-val versions: Map<String, String> by project
-val springBootVersion = versions["org.springframework.boot"]
+val springBootVersion = "2.7.18" // AutoConfiguration is added in 2.7.0, but can be used with older versions
```

Comment:
I don't really understand what this comment means

Reply:
AutoConfiguration is a class that we use to specify the order of autoconfig classes

even though it's added it 2.7.0, the tests run successfully with 2.6